students/tfbanks/Lesson09/Mailroom_pt5.py

```diff
@@ -0,0 +1,124 @@
+# Mailroom_pt5py Exercise by tfbanks
+
+# !/usr/bin/env python3
+
+import os
+# Donor Database containing names and donations
+
+
+
+class Donor:
+    def __init__(self, donations=None):
+        self.donations = donations if donations else []
+
+
+donor_data = {
+        'Tim Cooker': [1500.50, 25050.50, 15680.75],
+        'Elon Musket': [5250.25, 26500],
+        'Frank Petersmankempt': [550.60],
+        'Megan Morgan': [650.40, 1600, 20625.40],
+        'Marlene Wheeler': [820, 1222.80]} 
+
+
+class Donor_Actions:
+
+    def add_new_donor(donor, donations):
+        donor_data.setdefault((donor),[]).append(donations)  
```

Comment:
your functions should either be class methods 
```
def add_new_donor(self, donor, donations)
```
or staticmethods
```
@staticmethod
def add_new_donor(donor, donations)
```

here it isn't a proper use of class functions.